docs/processors/getting-started.mdx

```diff
@@ -27,7 +28,20 @@ needs to happen for different sources or destinations, you have two options:
 2. If records from some, but not all, sources (or destinations) need to be processed in the same way, then you need to
    create multiple processors (one for each source or destination) and configure them in the same way.
 
-## Adding and configuring a processor
+## Creating a processor through a pipeline configuration file
+
+Creating a processor through a pipeline configuration file can be done as below:
+```yaml
+- id: extract-name
+  type: extractfieldpayload
+  settings:
+    field: name
```

Comment:
Linking to the pipeline config docs sounds good to me. I put just the processor part because that's what I'd at least be interested in knowing, straight to the point. It's also analogous to the API example below. But I don't mind adding a whole pipeline either. Now that I think of it, I can probably just link to https://github.com/ConduitIO/conduit/blob/main/examples/processors/pipeline-extract-field-transform.yml.

Reply:
My goal is to let the user know where that processor part fits in. Similarly to how the [dead letter queue](https://conduit.io/docs/features/dead-letter-queue#configuring-a-dead-letter-queue) docs skip the irrelevant parts but still show you where the DLQ config is defined.

Maybe my example could be split into 2 examples, one for a connector processor and another for a pipeline processor, this would allow us to keep both very simple. For example:

```
version: 2.0
pipelines:
  - id: example-pipeline
    connectors:
      # define source and destination connectors
      # ...
    processors:
      - id: extract-name
        type: extractfieldpayload
        settings:
          field: name
```